openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -187,12 +187,11 @@ The following requirements apply for both, the Wallet and the Verifier, unless s
 ## ISO mdoc specific requirements for OpenID for Verifiable Presentations over W3C Digital Credentials API
 
 Requirements for both the Wallet and the Verifier:
-
-* The Credential Format Identifier MUST be `mso_mdoc`.
-* ISO mdoc Credential Format specific DCQL parameters as defined in Annex B.3.1 of [@!OIDF.OID4VP] MUST be used.
-* Verifier MAY request more than one Credential in the same request.
+* ISO mdoc Credential Format specific DCQL parameters, intent_to_retain and doctype as defined in Annex B.3.1 of [@!OIDF.OID4VP] MUST be present.
```

Comment:
Since `doctype` is now required in OpenID4VP i don't think the requirement is necessary anymore (and added backticks to intent_to_retain)

```suggestion
* ISO mdoc Credential Format specific DCQL parameter, `intent_to_retain` defined in Annex B.3.1 of [@!OIDF.OID4VP] MUST be present.
```